contravariant.cabal

```diff
@@ -65,8 +65,10 @@ library
   hs-source-dirs: src
   build-depends:
     base                              < 5,
-    transformers        >= 0.2 &&     < 0.6,
-    transformers-compat >= 0.3 &&     < 1
+    transformers        >= 0.2 &&     < 0.6
+
+  if impl(ghc <= 7.8)
```

Comment:
This should be `if !impl(ghc > 7.8)`. See previous PRs for explanations.

I think this is **important** as many look at edwards packages for an example. We have to keep high standard.